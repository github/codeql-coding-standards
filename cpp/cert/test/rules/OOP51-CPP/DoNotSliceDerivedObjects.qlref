rules/OOP51-CPP/DoNotSliceDerivedObjects.ql