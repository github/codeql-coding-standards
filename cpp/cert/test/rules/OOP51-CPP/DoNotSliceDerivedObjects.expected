| test.cpp:27:10:27:32 | call to Dog | Derived object sliced from type $@ to base $@. | test.cpp:9:7:9:15 | Chihuahua | Chihuahua | test.cpp:1:7:1:9 | Dog | Dog |
| test.cpp:28:11:28:13 | call to Dog | Derived object sliced from type $@ to base $@. | test.cpp:9:7:9:15 | Chihuahua | Chihuahua | test.cpp:1:7:1:9 | Dog | Dog |
| test.cpp:31:5:31:6 | call to Dog | Derived object sliced from type $@ to base $@. | test.cpp:9:7:9:15 | Chihuahua | Chihuahua | test.cpp:1:7:1:9 | Dog | Dog |
